components/match2/wikis/dota2/match_page_template.lua

```diff
@@ -118,71 +118,67 @@ return {
 				</div>
 			</div>
 			<h3>Player Performance</h3>
-			<div class="match-bm-lol-players-wrapper">
-				<div class="match-bm-lol-players-team"><div class="match-bm-lol-players-team-header">{{&opponents.1.iconDisplay}}</div>
+			<div class="match-bm-players-wrapper">
+				<div class="match-bm-players-team"><div class="match-bm-lol-players-team-header">{{&opponents.1.iconDisplay}}</div>
```

Comment:
Using `lol` class here, not sure if intended?

Reply:
I only removed `-lol` from classes that I needed to change, to keep the lol version in tact.